### PR TITLE
(Bug) Minted, Dutch: Whitelisted addresses are not initialized if list length > 60

### LINKED ISSUE #1052: (Bug) Minted, Dutch: Whitelisted addresses are not initialized if list length > 60
Steps:
1. Start creating Minted (or Dutch) crowdsale, decimals =18, 1 tier, whitelisted
2. In Step 3: add 61 whitelisted addresses from `*. csv` file:
https://drive.google.com/open?id=1XpOw_lqxZE6G5k0qAwWbhT2MTcTxFsWm
3. Complete deployment, check if `Register whitelisted addresses` transaction is skipped in Step 4

Expected result :
 - user should be able to create crowdsale with more than 61 whitelisted addresses
OR 
 -  should be restriction of length of whitelisted list  in Step 3

Actual result:
 - Whitelisted  addresses are not initialized if list length > 60 (for both strategy)

If you are reporting a problem with Token Wizard, please include the following information:

### Which network did you use? (Mainnet, Kovan, Rinkeby, etc.)

*Ropsten*
Proxy address: `0xbbe7f8062c2d6214af0451d8b4c3526ad88986ba`
ExecID: `0x116fea137db6e131133e7f2bab296045d8f41cc5607279db17b218cab0929a51 `

### If you were able to create it, what is the URL of your crowdsale?
http://localhost:3000/contribute?addr=0xbbe7f8062c2d6214af0451d8b4c3526ad88986ba&networkID=3

### Do you have screenshots showing the problem?

screen shot 2018-07-18 at 09 56 41
screen shot 2018-07-18 at 09 47 48

### Do you see errors in the dev console? If yes, please include a screenshot
No

---

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #1052 

The problem is also reproducible with values ​​lower than 60, and it is random
sometimes 59, sometimes 58, sometimes 60. I take the decision to put a limit of 50

- Add constant to limit whitelisted addresses to 50
- Add two new alerts for whitelisted addresses, when the validation is applied
- Refactor processWhitelist function
- Add function to validate whitelisted addresses length
- Apply whitelist validation to the component

### CODE REVIEW

**Comment by dennis00010011b** [comment]
@mariano-aguero 
Minted-Contribution page: Incorrect  maxCap for whitelisted investors
Ropsten, proxyID ` 0xce4c704c3af8f359929ed09d3fe3739913d5e263`
Steps:
 1. Create Minted crowdsale, 1 tier, duration by default, decimals=18, rate=1000, supply=100
 2. In Step 3 add whitelisted addresses from file :
 https://drive.google.com/open?id=1jGWohlqK3PttrPHT_Sma1ousATHSm0gz
 Private keys: https://drive.google.com/open?id=1QB8Y6qm11ZBRbPd9kiRF7ccd1_ALBJPN
3. Finish deployment
4. Set in wallet any whitelisted account
5. Open contribution file, observe max contibution, try to buy
Actual result:
 - whitelisted address can't buy
 - incorrect value of max contribution
screen shot 2018-07-31 at 12 35 21

**Comment by dennis00010011b** [comment]
Issue is filed https://github.com/poanetwork/token-wizard/issues/1067

File: src/utils/alerts.js [hunk 1]
```diff
@@ -290,7 +290,23 @@ export function noMoreReservedSlotAvailable() {
...
+    title: 'No more addresses available',
+    html: `You're not able to add more addresses to the whitelist. The maximum allowed is ${LIMIT_WHITELISTED_ADDRESSES}`,
+    type: 'info'
+  })
+}
+
+export function noMoreWhitelistedSlotAvailableCSV(count) {
+  return sweetAlert2({
+    title: 'You reach the limit of addresses',
+    html: `You're not able to add more addresses to the whitelist. Only ${count} addresses of the file could be added. The maximum allowed is ${LIMIT_WHITELISTED_ADDRESSES}`,
```
[the remark this answers is not in the document]
Comment (mariano-aguero) [reply src/utils/alerts.js]: Done

File: src/utils/alerts.js [hunk 2]
```diff
@@ -290,7 +290,23 @@ export function noMoreReservedSlotAvailable() {
...
     title: 'You reach the limit of reserved tokens',
-    html: `You're not able to reserve more tokens. Only ${count} reserved addresses of the file could be added`,
+    html: `You're not able to reserve more tokens. Only ${count} reserved addresses of the file could be added. The maximum allowed is ${LIMIT_RESERVED_ADDRESSES}`,
+    type: 'info'
+  })
+}
+
+export function noMoreWhitelistedSlotAvailable() {
+  return sweetAlert2({
+    title: 'No more addresses available',
```
[the remark this answers is not in the document]
Comment (mariano-aguero) [reply src/utils/alerts.js]: Done

**Pull request merged**